dist/api_data.py

```diff
@@ -454,29 +454,35 @@ def __cmp__(self, other):
             ]),
     Config('eviction_checkpoint_target', '5', r'''
         perform eviction at the beginning of checkpoints to bring the dirty
-        content in cache to this level, expressed as a percentage of the total
-        cache size.  Ignored if set to zero or \c in_memory is \c true''',
-        min=0, max=99),
+        content in cache to this level, absolute when greater than 100,
+        otherwise expressed as a percentage of the total cache size, should not
+        exceed the \c cache_size. Ignored if set to zero or \c in_memory is \c
+        true''',
+        min=0, max='10TB'),
```

Comment:
previously this parameter can max be specified as 99% of cache size, but now if user specifies as 100% error will not be returned, I think this is a change in behaviour, we need to return same error if users specifies as 100%, subsequent logic is also not handling this, I think this has to be taken care.

Reply:
maybe we can change wording and the behaviour to say "absolute when greater than or equal to 100" . so If you provide 100, it gets treated as 100 bytes? If I provide "1000", what unit is it treated? I guess that will be bytes.

Reply:
@bvpvamsikrishna  Thanks for bringing up. In my discussion about this with Alex, we thought that a value above 100 should be considered an absolute value. So I considered that 100 should be treated as percentage. I agree that with this change, if 100 is specified that would mean 100% and it won't give an error like earlier. 
@agorrod What would be your recommendation on this.
@sulabhM Yes, 1000 will have a unit of bytes but in current shape 100 would be 100%. I may make a change based on the final decision here.

Reply:
@nehakhatri5  May be I am not clear in my comment.  What I mean is the value of these parameters should not be same as cache size, instead they should be less than that, but I think in this logic we accept the value of eviction_target to be same as cache size.  for example, as a user if I specify cache_size and eviction target as "1 TB", both the values as same we will process and continue, but previously user will not be able to do this as max value accepted by eviction_target is 99 only.

Reply:
also if absolute value of eviction_target is greater than cache_size, how to detect and return error, for example if cache_size = 1 TB and eviction_target = 2 TB, do we return error ?